pkgs/applications/networking/browsers/chromium/common.nix

```diff
@@ -160,6 +160,10 @@ let
       ./patches/no-build-timestamps.patch
       # For bundling Widevine (DRM), might be replaceable via bundle_widevine_cdm=true in gnFlags:
       ./patches/widevine-79.patch
+      # for declarative extension installation
+      # (search for extensions at CHROMIUM_EXTENSION_DIRECTORY instead of 
+      # /usr/share/chromium/extensions)
+      ./patches/extension-search-path.patch
```

Comment:
Has this patch been sent upstream?

Reply:
No, I checked the source code of both chromium and ungoogled chromium, neither has this patch as far as I can tell.

Reply:
Upstreaming the patch would be best but I guess it's unlikely that upstream would accept it (and I'm not sure if they have a good place to ask for alternatives / discuss this).